ShacShifter/modules/WellFormedShape.py

```diff
@@ -0,0 +1,59 @@
+class WellFormedShape:
+    """The WellFormedShape class."""
+
+    def __init__(self):
+        self.classUri = ''
+        self.uri = ''
+        self.name = ''
+        self.description = ''
+        self.targetClass = []
+        self.targetNode = []
+        self.targetObjectsOf = []
+        self.targetSubjectsOf = []
+        self.nodeKind = ''
+        self.properties = []
+        self.defaultValue = ''
+        self.path = ''
+        self.closed = False
+        self.ignoredProperties = []
+        self.sOr = []
+        self.sNot = []
+        self.sAnd = []
+        self.sXone = []
+        self.message = {}
+        self.classes = []
+        self.dataType = ''
+        self.minCount = -1
+        self.maxCount = -1
+        self.minExclusive = -1
+        self.minInclusive = -1
+        self.maxExclusive = -1
+        self.MaxInclusive = -1
+        self.minLength = -1
+        self.maxLength = -1
+        self.pattern = ''
+        self.flags = ''
+        self.languageIn = []
+        self.uniqueLang = False
+        self.equals = []
+        self.disjoint = []
+        self.lessThan = []
+        self.lessThanOrEquals = []
+        self.nodes = []
+        self.qualifiedValueShape = ''
```

Comment:
I think there can exist more than one qValueShape. This may be also relevant for PropertyShape.qualifiedValueShape